go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -209,11 +209,23 @@ func (qre *QueryExecutor) Stream(callback func(*sqltypes.Result) error) error {
 		return err
 	}
 
-	conn, err := qre.getStreamConn()
-	if err != nil {
-		return err
+	// if we have a transaction id, let's use the txPool for this query
+	var conn *connpool.DBConn
+	if qre.transactionID != 0 {
+		txConn, err := qre.tsv.te.txPool.Get(qre.transactionID, "for streaming query")
```

Comment:
How do you plan to disable the transaction killer?

Reply:
Hmm... Not sure. I had not considered it, actually. Going to look at it now. If you have any suggestions, do share :)

Reply:
We use DBA workload which disables the sniper.